### ISSUE TITLE
Looking for parameters: l1_weight - gan_weight 

### ISSUE THREAD

**Issue by fabio-C** [phillipi/pix2pix#146]
In pix2pix [tensorflow implementation](https://github.com/affinelayer/pix2pix-tensorflow) I can set the following parameters:
--**l1_weight**: weight on L1 term for generator gradient
--**gan_weight**: weight on GAN term for generator gradient"

They are used to create the GAN loss, in this way:
`gen_loss = gen_loss_GAN * a.gan_weight + gen_loss_L1 * a.l1_weight`
Row 455 of [pix2pix.py](https://github.com/affinelayer/pix2pix-tensorflow/blob/master/pix2pix.py)

Now I am moving to Torch, but I can't find these parameters in this implementation. How can I modify it?  I didn't find also the same equation in this code.

**Comment by junyanz** [comment]
We only have one hyperparameter `lambda` in our implementation. You can consider `lambda=a.l1_weight / a.gan_weight`.

**Comment by fabio-C** [comment]
Thank you junyanz
So if I want an higher a.gan_weight contribution in GAN loss, I have to choose low lambda value, such as 0.1 ?

**Comment by junyanz** [comment]
Yes. I think the default lambda is 100. So you can use 1 or 10.
